src/App.tsx

```diff
@@ -1,73 +1,37 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Autocomplete } from './components/Autocomplete';
+import { Person } from './types/Person';
+
+const START_PHRASE = 'No selected person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handleSelected = useCallback((person: Person) => {
+    setSelectedPerson(person);
+  }, []);
+
+  const handleInputChange = (inputValue: string) => {
+    if (inputValue.trim() || inputValue.trim() === '') {
+      setSelectedPerson(null);
+    }
```

Comment:
if inputValue.trim() === '' then inputValue.trim() can't give you truthy value, this if doesn't make any sense

Reply:
But if delete the name immediately, state of the title doesn't change. -_-